daemon/src/maker.rs

```diff
@@ -4,23 +4,28 @@ use bdk::bitcoin::secp256k1::schnorrsig;
 use clap::Clap;
 use daemon::auth::{self, MAKER_USERNAME};
 use daemon::db::{self, load_all_cfds};
-use daemon::model::cfd::{Order, UpdateCfdProposals};
+use daemon::maker_cfd::{FromTaker, NewTakerOnline};
+use daemon::model::cfd::{Cfd, Order, UpdateCfdProposals};
 use daemon::model::WalletInfo;
+use daemon::oracle::Attestation;
 use daemon::seed::Seed;
 use daemon::wallet::Wallet;
 use daemon::{
     bitmex_price_feed, fan_out, housekeeping, logger, maker_cfd, maker_inc_connections, monitor,
     oracle, wallet_sync,
 };
+use futures::Future;
 use rocket::fairing::AdHoc;
 use rocket_db_pools::Database;
+use sqlx::SqlitePool;
 use std::collections::HashMap;
 use std::net::SocketAddr;
 use std::path::PathBuf;
 use std::str::FromStr;
 use std::task::Poll;
 use std::time::Duration;
-use tokio::sync::watch;
+use tokio::net::TcpListener;
+use tokio::sync::watch::{self, Receiver};
```

Comment:
IMHO it would be nicer if we didn't import `Receiver`, but rather qualify it, otherwise it's pretty hard to notice what kind of channel are we talking about.